ui/components/src/TimeRangeSelector/AbsoluteTimePicker.tsx

```diff
@@ -0,0 +1,149 @@
+      <Stack spacing={3} sx={{ padding: (theme) => theme.spacing(1, 2, 3) }}>
+        {showStartCalendar && (
+          <Box
+            sx={(theme) => ({
+              // TODO: create separate reusable calendar component
+              '.MuiPickerStaticWrapper-content': {
+                backgroundColor: theme.palette.background.default,
+              },
+              // reposition AM and PM buttons
+              '.MuiIconButton-sizeMedium': {
+                top: 80,
+                bottom: 'auto',
+                margin: theme.spacing(0, 3),
+              },
+            })}
+          >
+            <h3>Select Start Time</h3>
+            <StaticDateTimePicker
+              displayStaticWrapperAs="desktop"
+              openTo="day"
+              disableHighlightToday={true}
+              value={initialTimeRange.start}
+              onChange={(newValue) => {
+                if (newValue === null) return;
+                setTimeRange({ start: newValue, end: timeRange.end });
+              }}
+              onAccept={() => handleOnStartAccept()}
+              renderInput={(params) => <TextField {...params} />}
+            />
+          </Box>
+        )}
+        {showEndCalendar && (
+          <Box
+            sx={(theme) => ({
+              '.MuiPickerStaticWrapper-content': {
+                backgroundColor: theme.palette.background.default,
+              },
+              // reposition AM and PM buttons
+              '.MuiIconButton-sizeMedium': {
+                top: 80,
+                bottom: 'auto',
+                margin: theme.spacing(0, 3),
+              },
+            })}
+          >
+            <h3>Select End Time</h3>
+            <StaticDateTimePicker
+              displayStaticWrapperAs="desktop"
+              openTo="day"
+              disableHighlightToday={true}
+              value={initialTimeRange.end}
+              onChange={(newValue) => {
+                if (newValue === null) return;
+                setTimeRange({ start: timeRange.start, end: newValue });
+              }}
+              onAccept={() => handleOnEndAccept()}
+              renderInput={(params) => <TextField {...params} />}
+            />
+          </Box>
+        )}
+        <TextField
+          fullWidth
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+            // TODO: add helperText, fix validation after we decide on form state solution
+            const startDate = new Date(event.target.value);
+            const isValidDateRange = validateDateRange(startDate, timeRange.end);
+            if (isValidDateRange === true) {
+              const updatedRange = { start: startDate, end: timeRange.end };
+              setTimeRange(updatedRange);
+              onChange(updatedRange);
+            }
+          }}
+          value={format(timeRange.start, DATE_TIME_FORMAT)}
+          label="Start Time"
+          placeholder="mm/dd/yyyy hh:mm"
+          type="tel"
```

Comment:
Should this actually be `tel` since it's a date input?

Reply:
Yeah this is weird, I'll add a comment. Basically I inspected the regular MUI DateTimePicker props to build this StaticDateTimePicker and it actually uses 'tel' so that's what I landed on when 'date' was working as expected